src/editor/FluxQueryField.tsx

```diff
@@ -249,10 +248,15 @@ export default class FluxQueryField extends QueryField {
         .deleteForward(forward)
         .insertFragment(fragment)
         .moveForward(move)
-        .focus();
+        .focus()
```

Comment:
Any idea why we need to constantly `focus()`?

Reply:
None. This was the way it was before and so I didn't question it.